packages/modules/fsm/src/components/inbox/Filter.js

```diff
@@ -5,6 +5,7 @@ import { ApplyFilterBar } from "@egovernments/digit-ui-react-components";
 import { useTranslation } from "react-i18next";
 import Status from "./Status";
 import AssignedTo from "./AssignedTo";
+import Localities from "../../components/LocalityDropdown/Localities";
```

Comment:
move this to react component with extra prop boundary. So it can be used by all modules